app/scripts/controllers/user-storage/user-storage-controller.ts

```diff
@@ -27,15 +27,15 @@ export type UserStorageControllerState = {
   /**
    * Condition used by UI and to determine if we can use some of the User Storage methods.
    */
-  isProfileSyncingEnabled: boolean;
+  isProfileSyncingEnabled: boolean | null;
   /**
    * Loading state for the profile syncing update
    */
   isProfileSyncingUpdateLoading: boolean;
 };
 
 const defaultState: UserStorageControllerState = {
-  isProfileSyncingEnabled: true,
+  isProfileSyncingEnabled: null,
```

Comment:
Shouldn't this still be `true` for new users?

Reply:
WHOOPS. Yes this is correct, was too quick with it!